sorc/ncep_post.fd/SURFCE.f

```diff
@@ -371,7 +370,7 @@ SUBROUTINE SURFCE
                jj = jsta+j-1
                do i=1,iend-ista+1
                ii = ista+i-1
-                 datapd(i,j,cfld) = RHSFC(ii,jj)
+                 datapd(i,j,cfld) = max(H1,min(H100,RHSFC(ii,jj)))
```

Comment:
@SamuelTrahanNOAA Can you add a checking for undefined grids in calculation?
such as: if (RHSFC(ii,jj) /= spval)

Reply:
@SamuelTrahanNOAA Please add a change log.